js/data.js

```diff
@@ -1,4 +1,19 @@
 /* exported data */
 var data = {
-  films: []
+  view: 'list',
+  films: [],
+  reviews: [],
+  nextReviewId: 1,
+  editing: null
 };
+
+var previousData = localStorage.getItem('studios-ghibli-tab');
+if (previousData !== null) {
+  data = JSON.parse(previousData);
+}
+
+window.addEventListener('beforeunload', function (event) {
+  var reviewsJSON = JSON.stringify(data);
+  localStorage.setItem('studios-ghibli-tab', reviewsJSON);
+  this.localStorage.clear();
```

Comment:
Why are you deleting localStorage every time the page unloads?

Reply:
<img width="703" alt="Screen Shot 2022-08-04 at 10 39 55 AM" src="https://user-images.githubusercontent.com/97363006/182915716-b5d088ed-842d-4341-a95d-01aef6c29c8a.png">